language-translator/v2.js

```diff
@@ -38,7 +38,7 @@ function LanguageTranslatorV2(options) {
   // and correcting the default URL here will break older code, so it must be reserved for a major release.
   // todo: consider checking for options.url === LanguageTranslationV2.URL and also throw this warning then.
   // (This probably does't matter since the api didn't change)
-  if (!options.url) {
+  if (options && !options.url) {
```

Comment:
Make this `if (!options || !options.url) {` - we want to give the warning if the options object is omitted entirely.

Reply:
Ok, agreed, just made this change